.github/workflows/End2EndTestConfluentAws.yml

```diff
@@ -33,9 +33,23 @@ jobs:
         pip3 install protobuf
         pip3 install --upgrade snowflake-connector-python
         curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
-        sudo apt-get -y install jq
+        sudo apt-get -y install jq vim
         sudo apt-get -y install protobuf-compiler
 
+    - name: Install Squid as Proxy Server and Apache Utils for Password Authentication
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install squid3
+        sudo apt-get install apache2-utils
+
+    - name: Change squid config and run Proxy Server
+      run: |
+        sudo touch /etc/squid/passwords
+        sudo chmod 777 /etc/squid/passwords
+        sudo htpasswd -db -c /etc/squid/passwords admin test
+        sudo mv .github/scripts/squid.conf /etc/squid/squid.conf
+        sudo service squid start
```

Comment:
You also need to do this for those azure workflows.

Reply:
Yes, I will do it after one of this passes.